trackintel/analysis/__init__.py

```diff
@@ -1,3 +1,22 @@
-from .tracking_quality import *
-from .label import *
-from .modal_split import *
+from .tracking_quality import temporal_tracking_quality
```

Comment:
a generic question: 
why do we need the . before every `import` path?

Reply:
maybe see the discussion [here](https://stackoverflow.com/questions/14132789/relative-imports-for-the-billionth-time). 
`.` is the symbol for relative import relative to the current init.py file